packages/store-sync/src/recs/defineInternalComponents.ts

```diff
@@ -5,6 +5,7 @@ export type InternalComponents = ReturnType<typeof defineInternalComponents>;
 
 export function defineInternalComponents(world: World) {
   return {
+    /** @deprecated */
     RegisteredTables: defineComponent<{ table: Type.T }, Metadata, Table>(
```

Comment:
was torn on what to do with this! this is no longer needed as part of the storage adapter because we now use the `component.metadata.table` for things like schemas

I removed the populating of this table/component from the storage adapter and deprecated this (rather than removed) because it used to be part of the `components` return type of `syncToRecs`

I could just remove because it was considered "internal" and doesn't appear to be used: https://github.com/search?q=RegisteredTables&type=repositories

thoughts?

Reply:
i would be fine with removing